with-proxy/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '2'
+
+services:
+
+  nginx:
+    image: nginx
+    volumes:
+      - ./nginx:/etc/nginx:ro
+    networks:
+      - private
+
+  squid:
+    image: babim/squid
```

Comment:
It doesn't really matter, but I did discover Datadog publishes a Squid image too if you wanted to use that one: https://hub.docker.com/r/datadog/squid/

Reply:
Yeah, I found it as well, ask people if it's still maintained but apparently it was just a test that they did 🤷 So better not use it I think.

Reply:
Aha 😄 Agreed then.